.travis.yml

```diff
@@ -43,11 +43,10 @@ matrix:
         TOOLCHAIN=gcc-7-cxx17
         PROJECT_DIR=examples/Assimp
 
-    # FIXME: https://travis-ci.org/ingenue/hunter/jobs/313393099
-    # - os: linux
-    #   env: >
-    #     TOOLCHAIN=android-ndk-r17-api-24-arm64-v8a-clang-libcxx14
-    #     PROJECT_DIR=examples/Assimp
+    - os: linux
+      env: >
+        TOOLCHAIN=android-ndk-r17-api-24-arm64-v8a-clang-libcxx14
```

Comment:
If you want to enable broken toolchain then you should submit new version first:
- https://docs.hunter.sh/en/latest/creating-new/update.html#add-toolchains